Templating/Helper/DateTimeHelper.php

```diff
@@ -69,11 +71,11 @@ public function formatDate($date, $locale = null, $timezone = null, $dateType =
     }
 
     /**
-     * @param \DateTimeInterface|string|int $datetime
-     * @param null|string                   $locale
-     * @param null|string                   $timezone
-     * @param null|int                      $dateType See \IntlDateFormatter::getDateType
-     * @param null|int                      $timeType See \IntlDateFormatter::getTimeType
+     * @param \DateTime|\DateTimeInterface|string|int $datetime
```

Comment:
[Soullivaneuh suggested](https://github.com/sonata-project/SonataIntlBundle/pull/134/files/5570a04a221a36937b5ae6078b2e650c2d5b2871#r67526489) to add it because `DateTime` does not implement `DateTimeInterface` in PHP 5.3-5.4. What do you think?